dist/consumer.js

```diff
@@ -282,6 +288,9 @@ class Consumer extends events_1.EventEmitter {
                 .then(this.handleSqsResponse)
                 .catch((err) => {
                 this.emit('error', err);
+                if (isNonExistentQueueError(err)) {
+                    throw new Error(`Could not receive messages - non existent queue - ${this.queueUrl}`);
```

Comment:
note to self - maybe I should add try catch to my service for these cases?

Reply:
mm you don't have the queue names there so i don't think you can perform such validation